### ISSUE TITLE
some messages aren't displayed in the room

### ISSUE THREAD

**Issue by mvgorcum** [neilalexander/seaglass#120]
riot does show these messages, suggesting it is a room cache bug

**Comment by phildenhoff** [comment]
Hi @mvgorcum, can you provide some more details about what kind of messages aren't displayed? Do you mean some chat messages show up on riot but don't show up in Seaglass?

**Comment by mvgorcum** [comment]
Yes, for some reason seaglass wasn't showing some new messages that did show up in riot. However, after a restart the issue seems gone. I'll report back if the issue arises again

**Comment by mvgorcum** [comment]
Right, so after restarting the issue resurfaced, see attached screenshots.

<img width="1664" alt="Screenshot 2019-05-14 at 14 52 56" src="https://user-images.githubusercontent.com/24435423/57807219-03280c00-7761-11e9-9b46-d7818fe42181.png">
![Screenshot_20190514-145304_Riot im](https://user-images.githubusercontent.com/24435423/57807222-04593900-7761-11e9-9b6f-1dd676c00eb0.png)

**Comment by bobpaul** [comment]
**Steps to Reproduce**
1. Start a direct message with a friend
2. Go to a different matrix room
3. Receive a message in the DM from step 1. The message previews in the sidebar.
4. Switch to the DM from step 1, the message isn't in the room.
5. Receive another message in the DM (while viewing the room)
6. The message is received and shows in the room
7. Restart seaglass and view the DM.
8. All messages are present

**Expected Behavior**
It should not matter whether I'm looking at the room or now, Seaglass should process the events and display them in the correct room. With this bug, Seaglass isn't very usable.
